utils/wasm-builder/src/wasm_project.rs

```diff
@@ -246,8 +246,12 @@ fn create_project_cargo_toml(
 	// Add different profiles which are selected by setting `WASM_BUILD_TYPE`.
 	let mut release_profile = Table::new();
 	release_profile.insert("panic".into(), "abort".into());
-	release_profile.insert("lto".into(), true.into());
-	release_profile.insert("codegen-units".into(), 1.into());
+	release_profile.insert("lto".into(), "thin".into());
+
+	let mut production_profile = Table::new();
+	production_profile.insert("inherits".into(), "release".into());
+	production_profile.insert("lto".into(), "fat".into());
+	production_profile.insert("codegen-units".into(), 1.into());
```

Comment:
Is there a reason that we use `panic = "abort"` for both `release` and `dev` profiles but not for `production` profile? Defaults to `panic = "unwind"`.

Reply:
Offtopic: The tons of `.into()` calls kinda irk me, we should really think about changing the API of `Table::insert` to take a generic parameter `T: Into<String>`.

Reply:
`production` inherits from `release`. It uses `abort` implicitly.

Reply:
Then probably the Cargo documentation for `[profile.release]` defaults seems to be wrong:
https://doc.rust-lang.org/cargo/reference/profiles.html#release

Reply:
Ah so overriding `profile.release` also overrides all inherited profiles. Makes sense.